com.onesignal.unity.android/Editor/SetupSteps/ExportAndroidResourcesStep.cs

```diff
@@ -102,5 +114,10 @@ public override bool IsRequired
         
         private static readonly string _manifestPackagePath = Path.Combine(_pluginPackagePath, "AndroidManifest.xml");
         private static readonly string _manifestExportPath = Path.Combine(_pluginExportPath, "AndroidManifest.xml");
+
+        // OneSignalConfig name used in the 3.x version of the SDK
```

Comment:
While the name was primarily used in 3.x.x I think it is important to include this folder name was used in v5 as well. So maybe say old name used from 3.x.x to 5.0.2 in a comment.